modules/og_to_group/og_to_group.module

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Drupal\Component\Utility\Html;
+use Drupal\migrate\Plugin\MigrateSourceInterface;
+use Drupal\migrate\Plugin\MigrationInterface;
+use Drupal\migrate\Row;
+
+/**
+ * Implements hook_migrate_prepare_row().
+ *
+ * Get the Book page that is the old Group landing page and take it's alias and
+ * apply it to the new group alias.
+ */
+function og_to_group_migrate_prepare_row(Row $row, MigrateSourceInterface $source, MigrationInterface $migration) {
+  switch ($migration->id()) {
```

Comment:
why use a switch with one case?

Reply:
Copied from drupal.org documentation. I believe they showed using a switch as you might want to use the prepare row on multiple migrations. Performance should be the same using an if verses a switch.